README.md

```diff
@@ -99,6 +99,24 @@ OC_PASS=<password>
 3. `yarn run start`
 4. Navigate to <http://localhost:9000/example>
 
+### Running and updating the cluster as a service operator
+
+#### Install/update the operator
+
+1. Install `operator-sdk` (<https://sdk.operatorframework.io/docs/installation/>)
+2. `oc login` to the development cluster
+3. `oc project cluster-aas-operator`
+4. `operator-sdk run bundle quay.io/stolostron/cluster-templates-bundle:2.7.0-125bb4a8e82d8bce3e66ff4f083cf63e24292bbf --timeout 5m` (Use desired version from <https://quay.io/repository/stolostron/cluster-templates-bundle?tab=tags>)
```

Comment:
@rawagner 
How about having a latest image tag, and then it can just stay  quay.io/stolostron/cluster-templates-bundle:latest

Reply:
Sure, I can setup the latest tag too

Reply:
Updated the README to use `latest` https://github.com/stolostron/cluster-templates-console-plugin/pull/17